cocos/platform/CCFileUtils.cpp

```diff
@@ -1076,6 +1076,15 @@ void FileUtils::addSearchPath(const std::string &searchpath,const bool front)
         path += "/";
     }
 
+	auto iter = std::find(_searchPathArray.begin(), _searchPathArray.end(), searchpath);
+    if (iter != _searchPathArray.cend())
+    {
+        if (front)
```

Comment:
Is it better change to `if (front && iter != _searchPathArray.begin() )`?

Reply:
yes, I think so. I just applied the patch from the the previous approved commit.

Reply:
actually, no. It looks different. Actually I don't really know what front means. The check enters if a duplicate is found, and if front, or it will return.

Reply:
front means needs to add the path in the front of the array. So if it is already in the front, then don't have to remove and add again.